### PR TITLE
cloud: tikv/tiflash scale in

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -61,22 +61,14 @@ To decrease the number of TiDB nodes, take the following steps:
...
 3. Click **Modify** in the drop-down menu. The **Modify Cluster** page is displayed.
-4. On the **Modify Cluster** page, decrease the number of TiDB nodes.
+4. On the **Modify Cluster** page, decrease the number of TiDB, TiKV, or TiFlash nodes.
 5. Click **Confirm**.
 
-To decrease the number of TiKV or TiFlash nodes, you need to submit a support ticket. The PingCAP support team will contact you and complete the scaling within the agreed time.
-
 > **Warning:**
 >
 > Decreasing TiKV or TiFlash node number can be risky, which might lead to insufficient storage space, excessive CPU usage, or excessive memory usage on remaining nodes.
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:69]: @SunRunAway Should we keep the warning or remove it?

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -61,22 +61,14 @@ To decrease the number of TiDB nodes, take the following steps:
...
 
-To submit a support ticket, perform the steps in [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). For each node to be scaled, provide the following information in the **Description** box:
-
-- Cluster name: xxx
-- Cloud provider: GCP or AWS
-- Node type: TiKV or TiFlash
-- Current node number: xxx
-- Expected node number: xxx
+You can also increase the number of TiDB, TiKV, or TiFlash nodes using TiDB Cloud API through the [Modify a Dedicated Tier cluster](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster) endpoint. Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:71]:
```suggestion
You can also decrease the number of TiDB, TiKV, or TiFlash nodes using TiDB Cloud API through the [Modify a Dedicated Tier cluster](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster) endpoint. Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
```

**Review by qiancai — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

File: tidb-cloud/scale-tidb-cluster.md (hunk 1 above)
Comment (SunRunAway) [reply tidb-cloud/scale-tidb-cluster.md:69]: it's ok to keep it.

**Review by SunRunAway — APPROVED** [review]

**Pull request merged**